server/base/src/main/java/org/apache/accumulo/server/compaction/CompactionJobGenerator.java

```diff
@@ -162,23 +166,29 @@ public Map<String,String> getExecutionHints() {
     return dispatcher.dispatch(dispatchParams).getService();
   }
 
+  private Level getLevel(TableId tableId, CompactionServiceId serviceId,
```

Comment:
Curious if you can use the EscalatingLogger ( or some other variant of ConditionalLogger) that's included in #4558 .

Reply:
I can try that

Reply:
Updated to use EscalatingLogger in 5500afb.  Ran the new IT in this PR and looked at the logs and the changes are working.  I was wondering if the exception some of the log messages had would be included in the key created from arguments in ConditionalLogger.  Poked around a bit and found its not included.

Reply:
Do you think the exception needs to be in the Key?

Reply:
I'm wondering if there is a downside to having too many ConditionalLoggers, as there is a Caffeine cache for each one. I wonder if it would be better to have one and add the logger name to the key.